Console/Command/ThemeChangeCommand.php

```diff
@@ -87,14 +100,50 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             $scope = 'default';
         }
 
+        if (!is_numeric($scopeId)) {
+            switch ($scope) {
+                case 'websites':
+                    $scopeId = $this->getWebsiteId($scopeId);
+                    break;
+                case 'stores':
+                    $scopeId = $this->getStoreId($scopeId);
+                    break;
+            }
+        }
```

Comment:
The switch could be replaced by slightly nicer PHP8.0 match construct:

```suggestion
        if (!is_numeric($scopeId)) {
            $scopeId = match ($scope) {
                'websites' => $this->getWebsiteId($scopeId),
                'stores' => $this->getStoreId($scopeId),
                default => 0,
            };
        }
```
However, I was not sure whether you would like dropping PHP7 compatibility...

Reply:
No, let's keep PHP 7 compat in there (so disappointing but realistic)